en/docs/emr-connectivity/guides/emr-connector.md

```diff
@@ -83,4 +83,49 @@ You can further refer to the Epic Documentation on creating an [OAuth 2.0 App](h
     ```
 
 === "Micro Integrator"
-
+    As a Prerequsite [Setup Micro Integrator](../../install-and-setup/setup-mi-server.md)
+
```

Comment:
Shall we have a sentence about the overview of the sample scenario we are trying, starting something like below

"The following example demonstrates ....."